### ISSUE TITLE
Updating a model doesn't change date/file time (and some files can be corrupted)

### ISSUE THREAD

**Issue by ddozza** [camunda/camunda-modeler#719]
Modeler version 1.11.3 on Windows 10 

Case 1:
Open an existing model
Modify the model
Save the model
File date/time doesn't change

Case 2:
Create a new model
Save the model over an existing file
File date/time doesn't change and remain the same of the existing file

In case you have a system problem because of USB stick failure (when Win refuses to unmount the USB stick), some files result corrupted **even if the I've edited all the files on the harddisk** and I've saved it before closing the Modeler. 
It happens to me three times in the last two days.
  
Davide

**Comment by nikku** [comment]
Can you provide us with a model that allows us to reproduce this issue?

**Comment by ddozza** [comment]
I don't have a particular model to reproduce the issue. Modifying and
saving any  model doesn't change file time/date property.

About the case of corrupted files, I've not been using a USB key to save
or copy models but it happened that, working on Modeler and keeping a 
USB key inserted into the PC, my last models resulted corrupted after
reboot even if I saved them. It happened when trying to unmount the USB
key the operating system refused to do it. I forced the reboot and for
three times, open files  before the reboot, were filled of zeros. This
happened only for models files and not for Office files.

Davide


Il 22/01/2018 13:32, Nico Rehwaldt ha scritto:
